spanner/api/Program.cs

```diff
@@ -0,0 +1,989 @@
+                return 0;
+            }
+            else if (!string.IsNullOrEmpty(startTitle)
+                && !string.IsNullOrEmpty(endTitle))
+            {
+                // Both startTitle and endTitle provided, include them both in
+                // the method call to QueryDataWithIndexAsync.
+                response = Task.Run(async () =>
+                    await QueryDataWithStoringIndexAsync(
+                        projectId, instanceId, databaseId,
+                        startTitle, endTitle));
+            }
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);
+            Console.WriteLine($"Operation status: {response.Status}");
+            return 0;
+        }
+
+        public static object AddColumn(string projectId,
+string instanceId, string databaseId)
+        {
+            var response = Task.Run(async () =>
+            await AddColumnAsync(
+                projectId, instanceId, databaseId));
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);
+            Console.WriteLine($"Response status: {response.Status}");
+            return 0;
+        }
+
+        public static object WriteDataToNewColumn(string projectId,
+string instanceId, string databaseId)
+        {
+            var response = Task.Run(async () =>
+            await WriteDataToNewColumnAsync(
+                projectId, instanceId, databaseId));
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);
+            Console.WriteLine($"Response status: {response.Status}");
+            return 0;
+        }
+
+        public static object QueryNewColumn(string projectId,
+string instanceId, string databaseId)
+        {
+            var response = Task.Run(async () =>
+            await QueryNewColumnAsync(
+                projectId, instanceId, databaseId));
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);
+            Console.WriteLine($"Response status: {response.Status}");
+            return 0;
+        }
+
+        public static object QueryDataWithTransaction(string projectId,
+string instanceId, string databaseId)
+        {
+            var response = Task.Run(async () =>
+                await QueryDataWithTransactionAsync(
+                    projectId, instanceId, databaseId));
+            Console.WriteLine("Waiting for operation to complete...");
+            Task.WaitAll(response);
+            Console.WriteLine($"Operation status: {response.Status}");
+            return 0;
+        }
+
+        public static object ReadWriteWithTransaction(string projectId,
+            string instanceId, string databaseId)
+        {
+            try
+            {
+                int retryCount = 3;
+                var retryStrategy = new Incremental(retryCount, TimeSpan.FromSeconds(2), TimeSpan.FromSeconds(2));

```

Comment:
we normally use ExponentialBackoff for GCP services.  We should use that instead of Incremental.